client/src/redux/reducers/allocations.reducers.js

```diff
@@ -75,9 +81,23 @@ function allocations(state = initialState, action) {
         ...state,
         search: { ...state.search, results: action.payload.data }
       };
+    case 'ALLOCATION_OPERATION_REMOVE_USER_INIT': {
+      return {
+        ...state,
+        removingUserOperation: {
```

Comment:
Would this be better nested in an `operations` object, where we can include `addUser` as well?

Reply:
Yes, totally.  Would it be cool if I pick up the todo item ` Extend saga/reducer so that added user is added to list` and then do that there?

Reply:
Perfect!